### ISSUE TITLE
DnD-MagentoConnectorBundle options not availble when creating export-profile

### ISSUE THREAD

**Issue by dm-becker** [Agence-DnD/DnD-MagentoConnectorBundle#40]
hi everyone,  

i have a problem with the DnD-MagentoConnectorBundle.  
I installed the extension exactly as told in the README:
- I added the two necessary additions into the composer.json
- php composer.phar update
- And i added the bundles-line into the app/AppKernel.php

Unfortunately when i login into the akeneo account, and then click on "new export profile", it does not offer me the DnD-MagentoConnectorBundle Options.

Im not very familiar with composer yet, but what could maybe be a reason, is that the output-shell while updating composer does not mention the MagentoConnector Bundle. 

![installing_assets](https://cloud.githubusercontent.com/assets/9213884/8744077/a5f4c8b0-2c74-11e5-906a-74a9b113474d.png)

I would appreciate it, if someone could give me a clue where the error could be.

Thanks

**Comment by gregos** [comment]
only need to clear the cache : 

php app/console cache:clear -e prod

**Comment by DnD-Mimosa** [comment]
Hi @PeterBeckerDM,

Did you cleared the cache of your Akeneo PIM ? Then, were you able to see the differents exports in your interface ?

Regards,
Florian.

**Comment by gregos** [comment]
yes on the akeneo PIM server ,  you can clear the cache with this : 

php app/console cache:clear -e prod

**Comment by kuldeepemol** [comment]
I am not able to see the DnD Magento Connector Bundle in the export profile as well. I did remove the cache using php app/console cache:clear -e prod.

![screen shot 2017-10-31 at 2 37 53 pm](https://user-images.githubusercontent.com/33235785/32250287-1ee67fbc-be49-11e7-8e35-fd9e07710904.png)

**Comment by ghost** [comment]
Hello @kuldeepemol,

It seems you are using an Akeneo 1.6 or 1.7. 
Dnd Magento Connector Bundle is deprecated since the Enhanced Connector was released by Akeneo. 

Please use the Enhanced Connector to export your attributes and families and the native PIM export for others entities.

Regards,

Benjamin

**Comment by kuldeepemol** [comment]
Hello Benjamin,

Thank you for the prompt response. Can PimGento handle export done using Enhanced Connector and native PIM? If so, can this be automated? It would be great if you can point me in the right direction to automate the process of export (from Akeneo) and import (to Magento).

Best,
Kuldeep

**Comment by ghost** [comment]
Hello,

Yes PIMGento handle it. If you want to automate the process you need a middleware or a specific development to move exported files from your Akeneo server to your Magento server as Akeneo does not provide sftp export and PIMGento does not provide sftp import for csv files (but it's something we want to add in the future).

Regards,

Benjamin
